### DISCUSSION TITLE
General Doubt's

### DISCUSSION THREAD

**Discussion by Vikash-8090-Yadav** [Vikash-8090-Yadav/Future.WebNet#145] [Q&A]
## This Discussion is for the General  Talk's and Anyone can ask Doubt  related to Web3  , NFT  , Blockcain , Project Issue and Repo related in this . 
**Let's rock Together**  :rocket: :rocket:

**Comment by Vikash-8090-Yadav** [comment]
@Bharadwajshivam28  what  about the logo of the iwoc and kwoc and alll ... etc ... ? When you  will  raise the pr for that ?

**Comment by Bharadwajshivam28** [comment]
sir the pr have been raised also

**Comment by Bharadwajshivam28** [comment]
I guess you have merged it aslo

**Comment by Bharadwajshivam28** [comment]
if not then let me raise it again

**Comment by Bharadwajshivam28** [comment]
I have raised the pr again

**Comment by Vikash-8090-Yadav** [comment]
please let me know when u raised  the issue

**Comment by Bharadwajshivam28** [comment]
i have raised right now again

**Comment by Bharadwajshivam28** [comment]
sir i have deleted the old forked repo

**Comment by Bharadwajshivam28** [comment]
but i have created the pr

**Comment by Vikash-8090-Yadav** [comment]
don't rush much , raise the pr for the blog i ll merge it , then raise for the logo . do this  fast ..

**Comment by Bharadwajshivam28** [comment]
ok sir

**Comment by Bharadwajshivam28** [comment]
doing it

**Comment by Bharadwajshivam28** [comment]
@Vikash-8090-Yadav Blog has been merged now i am making pr of images
